hms-app/src/common/redux/hackathonSlice.ts

```diff
@@ -12,6 +12,7 @@ const initialState: HackathonState = {
     {
       id: '33a892ba-8a9e-4edf-947a-f421c86a14b5',
       title: '',
+      slug: '',
```

Comment:
on the backend an empty slug would be invalid (since we check that when we validate a hackathon).  You should give all of these values to ensure they are all valid

Reply:
the same thing is true about titles, an empty title is invalid so all of them should get titles as well